application.py

```diff
@@ -4,26 +4,15 @@
 import os
 
 import newrelic.agent  # See https://bit.ly/2xBVKBH
-import sentry_sdk
```

Comment:
Sentry doesn't work with 3.10 and we're not using it in prod anyway. Discussed IRL and decided to remove it from the app.